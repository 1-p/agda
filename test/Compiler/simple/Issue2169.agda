
module _ where

open import Common.Prelude

printB : Bool â†’ IO Unit
printB true  = putStrLn "true"
printB false = putStrLn "false"

_==_ = primFloatEquality

main : IO Unit
main =
  putStr "0.0 == -0.0 = " ,, printB (0.0 == -0.0) ,,
  return unit
