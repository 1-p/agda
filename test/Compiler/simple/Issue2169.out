EXECUTED_PROGRAM

ret > ExitSuccess
out > 0.0 == -0.0 = false
out >
